runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -10271,25 +10249,24 @@ genHeapAlloc(TR::Node * node, TR::Instruction *& iCursor, bool isVariableLen, TR
             iCursor = generateLoad32BitConstant(cg, node, allocSize, sizeReg, true, iCursor, conditions);
          }
 
-      TR::Register * lengthReg = NULL;
+      // Zero initialize newly allocated array or object if batch clearing is disabled and node can not skip zero initialization.
+      static bool disableBatchClear = feGetEnv("TR_DisableBatchClear") != NULL;
+      bool inlineZeroInitialization = disableBatchClear && !node->canSkipZeroInitialization();
+      TR::Register *sizePreserveRegister = NULL;
```

Comment:
sizePreserveRegister seems bit out of context (If I have recommended in last PR to rename it, I apologize), but lengthReg seems to be fine. No need to rename it.